CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 4.0.7
+
+### Features
+
+### Fixes
+- `[resources/laserfiche-base]` Trim existing strings of whitespace
+- `[resources/laserfiche-base]` rearranged words in a key to proper order

```

Comment:
you can say "changed key `DO_YOU_WANT_TO_YOUR_APPLY_FIELD_CHANGES` to `DO_YOU_WANT_TO_APPLY_YOUR_FIELD_CHANGES`"